### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by Trixz2018** [Sigmastorm7/MacroBlocks#3]
https://pastebin.com/EiTDietE

i dont how to handle this, i think this is with elvui and ur addon but idk. any help would be great

**Comment by Sigmastorm7** [comment]
I'll do some more digging to find out if there's a better solution but for now I managed to disable the Macro Frame skinning without any errors showing up.

The file you need to change is here is here:
`[WoW Install Directory]/_retail_/Interface/AddOns/ElvUI/Mainline/Modules/Skins/Macro.lua`

Open the file in any text editor and add this line to very top of the page above everything else:
`if true then return end`

Save the file and reload your UI. If any other graphical issues or errors show up you can just add them to this thread. I'm leaving it open for now.

**Comment by Trixz2018** [comment]
yeah i disabled the elvui macro skin ingame and it fixed the issue
